src/modules/fstab/main.py

```diff
@@ -152,20 +152,22 @@ def generate_crypttab_line_info(self, partition):
         if not mapper_name or not luks_uuid:
             return None
 
-        if mount_point == "/":
+        if mount_point != "/":
             return None
 
         return dict(
             name=mapper_name,
             device="UUID=" + luks_uuid,
             password="/crypto_keyfile.bin",
+            options="luks,keyscript=/bin/cat",
```

Comment:
I'm ok with this, though I don't fully understand what it's for. Is it necessary on Debian?